openslides_backend/action/motion_category/create_update_delete.py

```diff
@@ -0,0 +1,21 @@
+from ...models.motion_category import MotionCategory
+from ..action import register_action_set
+from ..action_set import ActionSet
+from ..default_schema import DefaultSchema
+
+
+@register_action_set("motion_category")
+class MotionCategoryActionSet(ActionSet):
+    """
+    Actions to create, update and delete motion categories.
+    """
+
+    model = MotionCategory()
+    create_schema = DefaultSchema(MotionCategory()).get_create_schema(
+        properties=["name", "prefix", "weight", "level", "meeting_id", "parent_id"],
+        required_properties=["name"],
+    )
+    update_schema = DefaultSchema(MotionCategory()).get_update_schema(
+        properties=["name", "prefix", "weight", "level", "meeting_id", "parent_id"]
```

Comment:
`parent_id`, `weight` and `level` should not be updateable. The sort view will handle it.

@jsangmeister Does this occur in other places in the code?

Reply:
parent_id, weight and level removed from the properties.